### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by NinjaCoderDev**
Fixed typos in different .md files

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
```
Comment (richardhand) [inline CODE_OF_CONDUCT.md]: @NinjaCoderDev, I would not make changes to this file as it's a standard template for Adobe open source projects. The originals in this file are actually okay.

File: content/src/content/jcr_root/apps/core/wcm/components/languagenavigation/v1/languagenavigation/README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ limitations under the License.
 -->
 Language Navigation (v1)
 ====
-Language Navigation component written in HTL that renders a global language structure navigation.
+Language Navigation component is written in HTL that renders a global language structure navigation.
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/languagenavigation/v1/languagenavigation/README.md]: @NinjaCoderDev - the previous description makes more sense than the change here, I would it as it was.

File: content/src/content/jcr_root/apps/core/wcm/components/navigation/v1/navigation/README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 Navigation (v1)
 ====
-Navigation component written in HTL that renders a website navigation tree.
+The navigation component is written in HTL that renders a website navigation tree.
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/navigation/v1/navigation/README.md]: same comment, leave the original.

File: content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md [hunk 4]
```diff
@@ -1,7 +1,136 @@
 <!--
 Copyright 2017 Adobe
 
+Licensed under the Apache License, Version 2.0 (the "License");<!--
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md]: There's a strange diff here, starting with the license.

File: content/src/content/jcr_root/apps/core/wcm/components/search/v1/search/README.md [hunk 5]
```diff
@@ -1,6 +1,95 @@
 <!--
+Copyright 2017 Adobe<!--
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/search/v1/search/README.md]: There's a strange diff here, starting with the license.

**Comment by richardhand** [comment]
@NinjaCoderDev, added a few comments, could you please also base your pull request against the `development` branch. Thanks!

**Comment by NinjaCoderDev** [comment]
Ok, closing this pull request and sending pull request to development branch with making the changes . Thanks :)

**Pull request closed without merging**